cv32e40x/tests/programs/custom/hpmcounter_basic_test/test.yaml

```diff
@@ -2,4 +2,5 @@ name: hpmcounter_basic_test
 uvm_test: uvmt_$(CV_CORE_LC)_firmware_test_c
 description: >
     HPM Counter Basic Test (donated by Arjan Bink)
+plusargs: >
```

Comment:
With the script handle this correctly?  (empty plusargs)

Reply:
Unresolved - I committed the wrong set of changes to my latest commit, do not merge until I have fixed this - did not intend to move the core hash for this commit.